Packs/SymantecDLP/Integrations/SymantecDLPV2/SymantecDLPV2.py

```diff
@@ -704,6 +851,202 @@ def get_list_remediation_status(client: Client) -> CommandResults:
     )
 
 
+def get_incident_original_message_command(client: Client, args: Dict[str, Any]):
+    """
+    Fetch the original message
+    """
+    try:
+        incident_id = args.get('incident_id')
+        results = client.get_incident_original_message_request(incident_id)
+        original_message_file = results.content
+
+        try:
+            original_filename = results.headers.get('Content-Disposition').split('=')[1]
+        except DemistoException as e:
+            original_filename = 'unknown'
+        demisto.results(fileResult(original_filename, original_message_file))
+
+        return CommandResults(
+            readable_output=f"Symantec DLP incident {incident_id} original message"
+        )
+    except DemistoException as e:
+        if '401' in str(e):
+            raise DemistoException(f"Error 401: Incident access not authorized or the incident does not exist. {e.res}")
+        else:
+            raise DemistoException(f"Error {e.res}")
```

Comment:
I've tweaked the way I return the file to the method you recommend. I have left the try-except block since I have provided additional information by catching the exception and checking for 401 errors that would help others understand why they are receiving in error. Furthermore, there is existing precedent in the integration to wrap the command functions in try-except blocks for this reason; see "get_incident_details_command" and "get_incident_details_fetch"

Reply:
it does not seem like you have changed anything in the updated version, please update code.